libs/payments/ui/src/styles/index.css

```diff
@@ -0,0 +1,3 @@
+.plan-details-item {
```

Comment:
Why does this not belong with the other styles in `apps/payments/next/app/styles/global.css`?